README.md

```diff
@@ -4,6 +4,7 @@ Formerly known as "Help to Heat".
 ## Initial Setup
 1. From the project root, run `cp envs/web.template envs/web`
 2. Populate the OS API key into the OS_API_KEY variable in `envs/web`. The key can be found in Keeper. Please only use a single value (i.e. `["my_key_here"]`), rather than all of the keys that are stored in Keeper, in order to avoid needing to rotate all of the keys if your local environment is accidentally leaked.
+3. Populate the Open EPC API token into the OPEN_EPC_API_TOKEN variable in `envs/web`. This key can be found in keeper.
```

Comment:
Applies to the above as well, but I suppose the bit about Keeper doesn't belong here as it's Softwire-specific.

Reply:
have removed